package.json

```diff
@@ -38,7 +38,7 @@
 		"tests-only": "tape test/*.js",
 		"pretest": "npm run lint",
 		"test": "npm run --silent tests-only",
-		"posttest": "npm run test:multirepo && npx npm@'>= 10.2' audit --production",
+		"posttest": "npm run test:multirepo && npx \"npm@>=10.2\" audit --omit=dev",
```

Comment:
i prefer keeping --production; it’s clearer semantically than omit dev, and it’ll never be removed since it’d break too much.

for the quotes, what is the exact problem? does windows not understand single quotes? Or is the fact that the entire argument isn’t quoted (only part of it) the problem? I use this posttest script on hundreds of other packages so I’d love to understand exactly why this fix works :-)

Reply:
cmd.exe only supports double quotes. I'll revert éther production flag change

Reply:
ah ok, so `npx npm@">=10.2"` would also work?

Reply:
I'll try again when I'm at my computer (going to ski now) but I think it doesn't work, possibly because of the `@` character

Reply:
thanks! totally fine to go with what you have here, but it's good to have the quirks documented/remembered :-)

Reply:
it works with the quotes at the same location